config/tileset/aerial.json

```diff
@@ -1133,11 +1133,11 @@
       "category": "Urban Aerial Photos"
     },
     {
-      "2193": "s3://linz-basemaps/2193/waimakariri_urban_2021_0-05m_RGB/01FKHMHNJ130J9DF3J9CSYZ41V",
-      "3857": "s3://linz-basemaps/3857/waimakariri_urban_2021_0-05m_RGB/01FKHMK3VK9GRFB762B8JW33HW",
-      "name": "waimakariri_urban_2021_0-05m_RGB",
+      "2193": "s3://linz-basemaps/2193/waimakariri_2023_0.1m/01H4Q66MSD5ATAS9QX25FZ385V",
+      "3857": "s3://linz-basemaps/3857/waimakariri_2023_0.1m/01H4Q681F7MWQY303BM8TKAT6K",
+      "name": "waimakariri_2023_0.1m",
```

Comment:
This needs to be inserted as a new block or the previous aerial imagery set won't be accessible in the Layer dropdown either. The previous configuration should be updated to zoom level 32 so that it is hidden in the Aerial basemap.